components/feature/findinpage/README.md

```diff
@@ -58,6 +58,38 @@ findInPageFeature.bind(session)
 
 🦊 A practical example of using feature find in page can be found in [Sample Browser](https://github.com/mozilla-mobile/android-components/tree/master/samples/browser).
 
+private object FindInPageItems {
+    const val PREVIOUS = "previous"
+    const val NEXT = "next"
+    const val CLOSE = "close"
+    const val OPEN = "open"
+    const val INPUT = "input"
+}
+
+internal fun emitOpenFact() = emitFindInPageFact(Action.CLICK, FindInPageItems.OPEN)
+internal fun emitCloseFact() = emitFindInPageFact(Action.CLICK, FindInPageItems.CLOSE)
+internal fun emitNextFact() = emitFindInPageFact(Action.CLICK, FindInPageItems.NEXT)
+internal fun emitPreviousFact() = emitFindInPageFact(Action.CLICK, FindInPageItems.PREVIOUS)
+internal fun emitCommitFact(value: String) =
+        emitFindInPageFact(Action.COMMIT, FindInPageItems.INPUT, value)
```

Comment:
Heh, that was probably not intended to be here? :)

Reply:
.. good catch